controllers/configurationpolicy_utils.go

```diff
@@ -175,6 +175,9 @@ func checkFieldsWithSort(mergedObj map[string]interface{}, oldObj map[string]int
 			// extra check to see if value is a byte value
 			mQty, err := apiRes.ParseQuantity(mVal)
 			if err != nil {
+				if oldObj[i] == nil {
+					oldObj[i] = ""
```

Comment:
Could you please assign this to a variable such as `oVal` to avoid modifying `oldObj`? I think this particular modification would *not* modify the map from the caller's perspective because `oldObj` isn't a pointer and `i` was not previously defined but it's a good practice to not modify the input arguments if the function/method name implies it is performing a read-only operation.